module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1862,6 +1885,7 @@ public function minify(Minified &$minified): void
     {
         $minified->ty = $this->getSearchType();
         $minified->cl = $this->getSearchClassId();
+        $minified->o = $this->getSearchOrigin();
```

Comment:
I'm not sure I understand why you would want to persist the search origin to the database. That seems like a session-level value rather than a permanent one. In other words, if I save a search, and then I revisit it a week later, it's going to feel strange for it to say "back to browse" results, because I haven't browsed recently. Similarly, if I click into a search result from my search history screen, I would no longer expect it to provide a "back to browse" link. It would make more sense to me if the browse results included a query parameter when linking to results, and that query parameter triggered the "back to browse" behavior, rather than persisting things so deeply.

Again, happy to discuss if you feel differently -- just my initial gut reaction.

Reply:
It makes sense, I can remove those changes
I initially did it that way, to have it being part of the link generated from the sid and integrated with the query
But it makes more sense without I guess